_posts/elements/2015-03-25-example-headings-new.md

```diff
@@ -0,0 +1,50 @@
+---
+layout:       element
+title:        Example - Demo Headings
+date:         2015-02-25 00:00:00
+
+categories:   elements
+tags:
+- example
+
+slug:         example-headings-new
+scssurl:      "https://github.com/edx/ux-pattern-library/blob/master/[path_to_source_sass_file].scss"
+
+description:  Examples of headings in a semi-real world context.
+
+info:  Vestibulum id ligula porta felis euismod semper. Donec id elit non mi porta gravida at eget metus. Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Etiam porta sem malesuada magna mollis euismod.
+---
+
+<section class="group">
```

Comment:
I'd prefer us to keep the markup as lean (just the elements and any supporting elements/classes/attributes as needed) as possible for these examples. Can you remove the ``<section>`` and ``<article>`` elements here?